Movie/MoviesMVP/MovieListViewController/MovieListViewController.swift

```diff
@@ -143,8 +112,18 @@ private extension MovieListViewController {
 
             movieSegmentControl.centerXAnchor.constraint(equalTo: view.centerXAnchor),
             movieSegmentControl.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
-            movieSegmentControl.widthAnchor.constraint(equalToConstant: 250),
-            movieSegmentControl.heightAnchor.constraint(equalToConstant: 25)
+            movieSegmentControl.widthAnchor.constraint(equalToConstant: Constants.segmentControlWidth),
+            movieSegmentControl.heightAnchor.constraint(equalToConstant: Constants.segmentControlHeight)
         ])
     }
 }
+
+// MARK: - MovieListViewProtocol
+
+extension MovieListViewController: MovieListViewProtocol {
+    func updateView() {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
```suggestion
        DispatchQueue.main.async {  in
```
на DispatchQueue [weak self] не распространяется